### PR TITLE
refactor: remove global.renovateCache

### PR DESCRIPTION
**Pull request by rarkins**

### CODE REVIEW

File: lib/util/cache/global/file.ts [hunk 1]
```diff
@@ -7,16 +7,19 @@ function getKey(namespace: string, key: string): string {
   return `${namespace}-${key}`;
 }
 
-let renovateCache: string;
+let cacheFileName: string;
```
Comment (rarkins) [inline lib/util/cache/global/file.ts:10]: This is unrelated the main refactoring but I felt it deserved a more accurate variable name

File: lib/util/cache/global/file.ts [hunk 2]
```diff
@@ -7,16 +7,19 @@ function getKey(namespace: string, key: string): string {
...
 }
 
-async function get<T = never>(namespace: string, key: string): Promise<T> {
+export async function get<T = never>(
+  namespace: string,
+  key: string
+): Promise<T> {
   try {
-    const res = await cacache.get(renovateCache, getKey(namespace, key));
+    const res = await cacache.get(cacheFileName, getKey(namespace, key));
```
Comment (viceice) [inline lib/util/cache/global/file.ts:22]: Do we need to check for initialized cacheFileName?

File: lib/util/cache/global/redis.ts [hunk 3]
```diff
@@ -17,12 +17,15 @@ export function end(): void {
   }
 }
 
-async function rm(namespace: string, key: string): Promise<void> {
+export async function rm(namespace: string, key: string): Promise<void> {
```
Comment (viceice) [inline lib/util/cache/global/redis.ts:20]: Is it for testing?

File: lib/util/cache/global/redis.ts [hunk 4]
```diff
@@ -22,7 +22,13 @@ async function rm(namespace: string, key: string): Promise<void> {
   await client?.del(getKey(namespace, key));
 }
 
-async function get<T = never>(namespace: string, key: string): Promise<T> {
+export async function get<T = never>(
+  namespace: string,
+  key: string
+): Promise<T> {
+  if (!client) {
```
Comment (viceice) [inline lib/util/cache/global/redis.ts:29]: We don't need this check, as we use the question operator later.

Comment (rarkins) [reply lib/util/cache/global/redis.ts:29]: Let's go with explicit rather than clever. I'll remove the later check

**Review by viceice — APPROVED** [review]

**Pull request merged**